src/app/components/editor/index.js

```diff
@@ -5,19 +5,25 @@ import {
 	BlockEditorKeyboardShortcuts,
 	WritingFlow,
 	ObserveTyping,
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__unstableEditorStyles as EditorStyles,
 } from '@wordpress/block-editor';
-import {
-	SlotFillProvider,
-	DropZoneProvider,
-	Popover,
-} from '@wordpress/components';
+import { SlotFillProvider, Popover } from '@wordpress/components';
+import { getRegisteredThemes } from '../../api';
+import { useTheme } from '../../local-storage';
 
 export function Editor( { initialBlocks } ) {
 	const [ blocks, setBlocks ] = useState( initialBlocks );
+	const [ currentThemeName ] = useTheme();
+	const currentTheme =
+		getRegisteredThemes().find(
+			( theme ) => theme.name === currentThemeName
+		) || getRegisteredThemes()[ 0 ];
```

Comment:
It looks like the logic to retrieve the theme object is something that we duplicate a lot across files, might be good to move to a reusable hook at some point.